xbmc/filesystem/BlurayDirectory.cpp

```diff
@@ -291,4 +296,57 @@ std::string CBlurayDirectory::HexToString(const uint8_t *buf, int count)
   return std::string(std::begin(tmp), std::end(tmp));
 }
 
+std::vector<BLURAY_TITLE_INFO*> CBlurayDirectory::GetUserPlaylists()
+{
+  std::string root = m_url.GetHostName();
+  std::string discInfPath = URIUtils::AddFileToFolder(root, "disc.inf");
+  std::vector<BLURAY_TITLE_INFO*> userTitles;
+  CFile file;
+  char buffer[1025];
+
+  if (file.Open(discInfPath))
+  {
+    CLog::Log(LOGDEBUG, "CBlurayDirectory::GetTitles - disc.inf found");
+
+    CRegExp pl(true);
+    if (!pl.RegComp("(\\d+)"))
+    {
+      file.Close();
+      return userTitles;
+    }
+
+    uint8_t maxLines = 100;
+    while ((maxLines > 0) && file.ReadString(buffer, 1024))
+    {
+      maxLines--;
+      if (StringUtils::StartsWithNoCase(buffer, "playlists"))
+      {
+        int pos = 0;
+        while ((pos = pl.RegFind(buffer, static_cast<uint32_t>(pos))) >= 0)
```

Comment:
at least replace `uint32_t` with `unsigned int`, which is what RegFind signature uses